packages/neuron-wallet/src/block-sync-renderer/sync/indexer-connector.ts

```diff
@@ -19,22 +19,22 @@ export interface LumosCellQuery {
 }
 
 export interface LumosCell {
-  block_hash: string
-  out_point: {
-    tx_hash: string
+  blockHash: string
```

Comment:
Is there a `lumos cell` type defined in `lumos`?

Reply:
the `Cell` in lumos is a bit different from Neuron's `LumosCell`

```ts
// Neuron's LumosCell
type LumosCell = Required<Cell>
```

Reply:
Gotcha